pkg/apiserver/apis/datapackaging/types.go

```diff
@@ -127,3 +132,10 @@ type ValuesSchema struct {
 	// +kubebuilder:pruning:PreserveUnknownFields
 	OpenAPIv3 runtime.RawExtension `json:"openAPIv3,omitempty"`
 }
+
+// IncludedSoftware contains the underlying Software Contents of a Package
+type IncludedSoftware struct {
+	Name        string `json:"name,omitempty" protobuf:"bytes,1,opt,name=name"`
+	Version     string `json:"version,omitempty" protobuf:"bytes,2,opt,name=version"`
```

Comment:
i wonder if we can highlight that this version is a "third party version" somehow. eg doesnt have to follow strict semver format Package's spec.version needs to follow

Reply:
Not that I can think of - not semantically through a name.

Reply:
Any follow ups on this suggestion @cppforlife before I go ahead and mark this as ready to go without any changes to the `Version` field?

Reply:
the thing that we wish we could convey through the name, i want to make sure i understand it:  from the package's perspective, this version field is "theirs" and not "ours" ?

Reply:
if so,  options could include
- TheirVersion
- UnderlyingVersion
- PackagedVersion

(of those i'd vote for the last one,  but 🤷 )

Reply:
I personally think that Version is what users will understand as it relates to the Name of the IncludedSoftware. And to be honest, this is the one name I would change (but 🤷‍♂️ )